### PR TITLE
add update and details features

### PR DESCRIPTION
**Pull request by lluminita**
solves issues: #45 #44 #43 

Former closed PR: https://github.com/lluminita/codeeve/pull/46

### CODE REVIEW

File: projects/templates/projects/project_details.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                <a href="/projects/">Go to list all projects</a>
+                <a href="update/">Update the project</a>
+                <a href="delete/">Delete the project</a>
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">{{ project.title }}</h2>
+                    <p>{{ project.created_at|date:"D d M Y"  }}</p>
+                    <div
+                      {% if project.difficulty == "beginner" %}
```
Comment (mei-li) [inline projects/templates/projects/project_details.html]: I think you can do
`<div class=glossy-{{ project.difficulty }}>`

File: projects/templates/projects/update_project.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+{% block content %}
+    <div class="content-section-a">
+
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <form action="{% url 'project_details' pk=project.pk %}" method="post" enctype="multipart/form-data">
```
Comment (mei-li) [inline projects/templates/projects/update_project.html]: btw it says here https://docs.djangoproject.com/en/1.9/topics/http/file-uploads/ to check with security on uploading, we should do it when deploying. Maybe heroku has it already though

**Comment by lluminita** [comment]
I fixed the _difficulty_ comment and I remove the create and update templates and replaced with project_form, this way there is no need to use the attribute template_name in the views.

**Comment by mei-li** [comment]
I made this issue for the owner https://github.com/lluminita/codeeve/issues/52

**Comment by mei-li** [comment]
:+1:

**Pull request merged**